### DISCUSSION TITLE
I Use the latest template, New up a .Net 8 Uno App - Can't get SKXamlCanvas to work

### DISCUSSION THREAD

**Discussion by bitzeal-johan** [unoplatform/uno#14666] [General]
The only thing I modified in the project was a slight modification to the MainPage.Xaml, adding:

`<windows:SKXamlCanvas></windows:SKXamlCanvas>`

(and of course ` xmlns:windows="using:SkiaSharp.Views.Windows"_`)

I get the following error when I try to build for Windows:

1>C:\repo\SkieaLabs\SkieaLabs\SkieaLabs\MainPage.xaml : Xaml Internal Error error WMC9999: Type universe cannot resolve assembly: Uno.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null.

**Comment by bitzeal-johan** [comment]
Ok so after spending a day trying different combinations of nuget packages in different platform- and / or shared projects, what you need to do to get past the build issue is:

To the SHARED project (the one without any platform extension) add:
SkiaSharp.Views.Uno.WinUI
AND
SkiaSharp.Views.WinUI

I love Uno platform and really appreciate the work that people put into it. That said it would be invaluable if the developers just dropped what they were doing for a day and made sure that obsolete documentation, youtube videos and templates etc are removed, and that the documentation that reflects the current state of Uno platform is comprehensive and up to date. Just do an internet search on whatever you are working on and make sure information is correct and usable (i.e. someone without Uno experience can use the information to get something running)

**Reply by jeromelaban** [reply]
Thanks for your comment. 

Updating older documentation can be out of our control, unfortunately, yet we're trying our best to document all scenarios that we know of and keep them up to date. The help from our community is invaluable here.

Know that you may find our latest samples here: https://github.com/unoplatform/uno.samples.

For SkiaSharp, specifically, a google search pointed me to https://github.com/unoplatform/uno/discussions/11080 which gives information you were looking for, even if it's not our main docs. 

What topics did you find that is not relevant? If you have a list, please let us know.
